api/opencl_runtime_layer.asciidoc

```diff
@@ -5427,6 +5427,23 @@ handle is used by an OpenCL command queued to a command-queue without being
 acquired.
 This is to guarantee that the state of the memory objects is up-to-date and
 they are accessible to OpenCL.
+
+The following restrictions shall apply -
+  * Each memory object must be acquired only once. Acquiring memory object 
+    multiple times without releasing it may result in implementation defined
```

Comment:
s/may result/results/
s/implementation defined/implementation-defined/ (we tend to hyphenate this in the rest of the spec)